components/mpas-ocean/src/Registry.xml

```diff
@@ -984,11 +984,15 @@
 		<nml_option name="config_frazil_heat_of_fusion" type="real" default_value="3.34e5" units="J kg^-1"
 					description="Energy per kilogram released when sea water freezes. NOTE: test and make consistent with E3SM."
 					possible_values="Any positive real number."
-		/>
+		/> 
 		<nml_option name="config_frazil_ice_density" type="real" default_value="1000.0" units="kg m^-3"
 					description="Assumed density of frazil. NOTE: test and make consistent with E3SM."
 					possible_values="Any positive real number."
 		/>
+		<nml_option name="config_frazil_ice_porosity" type="real" default_value="-1.0" units="1"
+					description="Internal porosity of frazil"
+					possible_values="Positive real number from 0 to 1. Negative value reverts to using the reference salinity for internal frazil."
```

Comment:
```suggestion
					possible_values="Positive real number from 0 to 1. Indicates the fraction of local ocean salinity to use for frazil ice. If non-negative, applies to both land ice frazil and sea ice frazil. Negative value reverts to using the reference salinity for internal frazil and 0 for land ice.
```

Also modify the description of `config_frazil_sea_ice_reference_salinity` so that it does not say that 0 salinity is always used for land ice.

Reply:
@njeffery, I feel uncomfortable about this approach of using a negative value as a flag.  This is not an approach we have taken elsewhere in MPAS-Ocean.  Instead, we have, at least as far as I'm aware, always used a separate logical to turn the feature on or off.

Reply:
+1 to this point by @xylar

Reply:
@xylar : that's a good point. I figured this would be a clean option because this is not really an independent feature.  It just makes explicit the choice to use either a fixed reference salinity or a fraction of the local salinity  in the internal frazil calculation.

Reply:
@njeffery, I take your point that it's a little different than a typical on/off feature.  In such cases, we typically have a string namelist option that selects between the possible features and then we have one or more logicals in the code that determine the behavior (to avoid constantly performing string comparisons except at init within the code itself).  Would that be an approach you could get behind here as well?

Reply:
Sure.  I'll try to get to it this week.